apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx

```diff
@@ -22,10 +23,10 @@ export const ManualOverride = (props: AnyAction) => {
   )
 }
 
-function buildOptions(_type: ActionType) {
+function buildOptions(type: ActionType) {
   // What states can we go to?
-  // Assume eTransfer for now
-  const states = Object.keys(graph);
+  // We may need to consider a cleaner view than the raw to-any-state
+  const states = type == "Buy" ? Object.keys(etransfer) : Object.keys(graph);
```

Comment:
does not consider manual purchases, as they will probably never fail (can implement it if it ever proves necessary)

Reply:
"will never fail" makes me really anxious every times I heard that one it failed at some point

Reply:
you're right.  Implementing that now